spec/features/transfer_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+feature 'time transfer' do
+  let(:user) do
+    Fabricate(:user, email: 'user@timeoverflow.org', password: 'papapa22')
+  end
+  let(:other_user) do
+    Fabricate(:user, email: 'other_user@timeoverflow.org', password: 'papapa22')
+  end
+  let(:organization) { Fabricate(:organization) }
+
+  before do
+    # Create terms and conditions
+    Document.create(label: "t&c") do |doc|
+      doc.title = "Terms and Conditions"
+      doc.content = "blah blah blah"
+    end
+
+    user.add_to_organization(organization)
+    other_user.add_to_organization(organization)
+  end
+
+  it 'transfers time from one account to another' do
+    offer = Fabricate(:offer, user: other_user, organization: organization)
+
+    sign_in_with(user.email, 'papapa22')
+    navigate_to_member
+    navigate_to_transfer_for(offer)
+    submit_transfer_form_with(hours: 2)
+
+    expect(page).to have_css('.transactions tr', count: 1, text: '2:00')
+  end
+
+  def submit_transfer_form_with(hours: nil, minutes: nil)
+    within transfer_form do
+      fill_in 'transfer_hours', with: hours
+      fill_in 'transfer_minutes', with: minutes
+
+      # hack alert! there is no translation for this string. How we build the
+      # copy then?
```

Comment:
I'd rather create an issue

Reply:
this is just active record magic, [see here](https://github.com/coopdevs/timeoverflow/blob/6d069bcf46a234fe8a65ed7955f6f02ffb0c4846/app/views/transfers/new.html.erb#L39)